src/App.tsx

```diff
@@ -15,49 +17,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  none = '',
+  alphabet = 'alphabet',
+  length = 'length',
+}
```

Comment:
```suggestion
enum SortType {
  None = '',
  Alphabet = 'alphabet',
  Length = 'length',
}
```
CamelCase for enum keys